networks.json

```
@@ -105,6 +125,10 @@
     "42161": {
       "address": "0x412c36e5011cd2517016d243a2dfb37f73a242e7",
       "transactionHash": "0x93085a71063a95b11fe8860c17ace8d3836895a8647bf1b6f639e2f6c9d282bc"
+    },
+    "232": {
+      "address": "0x412c36e5011CD2517016D243a2dfB37f73A242E7",
```

Also consistent capitalization:

```suggestion
      "address": "0x412c36e5011cd2517016d243a2dfb37f73a242e7",
```